packages/cycle-scripts/configs/webpack.config.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+// Silence webpack2 deprecation warnings
+// https://github.com/vuejs/vue-loader/issues/666
+process.noDeprecation = true
+
+const { createConfig, defineConstants, env, entryPoint, setOutput, sourceMaps, addPlugins } = require('@webpack-blocks/webpack2');
+const babel = require('@webpack-blocks/babel6');
+const devServer = require('@webpack-blocks/dev-server2');
+const postcss = require('@webpack-blocks/postcss');
+const sass = require('@webpack-blocks/sass');
+const typescript = require('@webpack-blocks/typescript');
+const tslint = require('@webpack-blocks/tslint');
+const extractText = require('@webpack-blocks/extract-text2');
+const autoprefixer = require('autoprefixer');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const ProgressBarPlugin = require('progress-bar-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const BabiliPlugin = require('babili-webpack-plugin');
+const path = require('path');
+
+const babelConfig = {
```

Comment:
what will be shared the babel config or the babel-loader config? as in there there are things that are pure babel-loader for webpack related as `cacheDirectory: true`. Perhpash we can make this more evident and with better naming of what get shared and what not? Otherwise I think we'll be sharing with ts stuff that he doesn't need/support and I'll rather avoid that